app/frontend/src/components/Navigation/FooterNav.tsx

```diff
@@ -19,7 +19,7 @@ function FooterNav({ menu, logoDesktop, logoMobile }: FooterNavProps) {
   const Logo = () => {
     return (
       <div className="footer-icons-on-dark">
-        <a href="/">
+        <a className='logo-anchor' href="/">
```

Comment:
This looks good so feel free to merge in your changes in at any time. One thing I do want to note is that I don't believe the outer <div> is doing anything valuable. If this works, I think this could be a bit cleaner and save us some css-namespaces:

```
return (
<a className="footer-icons-on-dark">
...
</a>
)
```

Reply:
Thanks @Aveline-art! You are right—the outer div is unnecessary so I removed it and cleaned up the classes by deleting `.logo-anchor` and adding the `display: block;` declaration to `.footer-icons-on-dark`. Local changes passed my testing on Chrom, Safari and Firefox so I pushed the new commits.